src/Parser/DemonHunter/Havoc/Modules/Features/AlwaysBeCasting.js

```diff
@@ -20,16 +20,24 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
     SPELLS.DEATH_SWEEP.id,
     SPELLS.EYE_BEAM.id,
     SPELLS.FURY_OF_THE_ILLIDARI.id,
-    SPELLS.METAMORPHOSIS_HAVOC.id,
     SPELLS.THROW_GLAIVE_HAVOC.id,
     SPELLS.CHAOS_NOVA.id,
 
     // Talents
     SPELLS.FELBLADE_TALENT.id,
     SPELLS.FEL_BARRAGE_TALENT.id,
     SPELLS.FEL_ERUPTION_TALENT.id,
+
+    //Utility
+    SPELLS.FEL_RUSH.id,
+    SPELLS.VENGEFUL_RETREAT.id,
   ];
 
+  static STATIC_GCD_ABILITIES = {
+    [SPELLS.FEL_RUSH]: 250,
```

Comment:
I assume this is also not a real GCD, but the animation time?

Reply:
http://www.wowhead.com/spell=195072/fel-rush
Real gcd in fact vs 

http://www.wowhead.com/spell=198793/vengeful-retreat

Which is off the gcd but has a 1s duration